controllers/controllers.js

```diff
@@ -27,7 +27,28 @@ exports.getArticleByID = (req, res, next) =>{
 
 exports.getAllArticles = (req, res, next) =>{
     return fetchAllArticles().then((queryResponse)=>{
-        console.log(queryResponse)
         res.status(200).send({articles: queryResponse})
     })
+}
+
+exports.getArticleComments = (req, res, next) =>{
+   const doesArticleExist= checkArticleExists(req.params.article_id)
+
+   const retrieveComments = fetchArticleComments(req.params.article_id)
+  
+
+   return Promise.all([doesArticleExist, retrieveComments]).then((queryResponse)=>{
+        const [,comments] = queryResponse
```

Comment:
Nice logic. Since you've used the comma here, it shows that you are looking for the second thing in the array, it might be easier to just swap the promises around in the Promise.all and then access the first one.